src/main/java/frc/robot/Constants.java

```diff
@@ -16,6 +16,8 @@
 public final class Constants {
   public static final double SIMPLE_AUTON_SPEED = 0.2; // PLACEHOLDER
   public static final double SIMPLE_AUTON_RUNTIME = 5.0; // PLACEHOLDER
+  public static final double OUTTAKE_SPEED = 5.0; // PLACEHOLDER
```

Comment:
Speed constant should go in a new nested class for outtake constants. Motor ID constant should be more clearly labeled and included in the nested class as well.